### DISCUSSION TITLE
Ban recipes

### DISCUSSION THREAD

**Discussion by HomeLandingsDE** [Zrips/CMI#20] [Feature Requests]
Can you ban recipes?
Example a player should not be able to make weapons so the recipe is banned and the player can not craft it.

Is that possible?

**Comment by HomeLandingsDE** [comment]
Push

**Comment by MCTyler** [comment]
You can do this in WorldGuard, it's complicated though.

**Comment by JohOply** [comment]
Does the issue refer to vanilla recipes? (if not, i will create a new ticket)

**Comment by mrfloris** [comment]
Personally I think this would be quite a nice addition. I understand this could become quite extensive. But I hope it becomes a feature.

Adding the option to identify a recipe and negate a (custom) recipe like we can do with Enchantments would help.
Players from the default group would be able to craft iron armor, but require to rankup in order to craft diamond armor.
At the least we would be able on a (hardcore) server to prevent crafting buckets, shears, or a custom recipe (unless you are admin).
